### PR TITLE
Migrate advanced search tests

### PR DESCRIPTION
**Pull request by karanh37**
### Describe your changes:

Fixes <issue-number>

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/playwright.config.ts [hunk 1]
```diff
@@ -64,23 +64,30 @@ export default defineConfig({
       name: 'setup',
       testMatch: '**/*.setup.ts',
     },
+    // {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/playwright.config.ts:67]: @karanh37,is this change needed?

**Review by chirag-madlani — APPROVED** [review]
